### ISSUE TITLE
[FEATURE] Able to autogenerate content for categories

### ISSUE THREAD

**Issue by LS-Myron** [mage-os-lab/module-catalog-data-ai#5]

**Comment by kuafucode** [comment]
Categories tend to be in limited size and static, even if users do want AI generated category content, it won't take them much copy pasting, or export importing to fulfill the one time need.

**Comment by LS-Myron** [comment]
Yes that is correct but if you have a button added to the description field (https://github.com/mage-os-lab/module-catalog-data-ai/issues/4) then the user could add its own keywords in the description field allowing them to make AI generate content on the fly.
